modules/ui/rapid_internal_features_storage.js

```diff
@@ -0,0 +1,54 @@
+var memStorage = {}; 
+
+
+export function rapidInternalFeaturesStorage() {
+    // Cloned from context, but we need storage from within the actions
+    // modules which are context-free. 
```

Comment:
Ah, I was hoping to be able to make this part of `rapid_context.js`, but it makes sense that these values are "context-free" and done at build-time.